.gitignore

```diff
@@ -0,0 +1 @@
+cmd/scouting-backend/scouting-backend
```

Comment:
This won't ignore Windows executables. I think we need a second ignore directive for them.

Reply:
Also let's add a leading slash so it only matches this path from the root directory.